src/Common/src/Interop/Unix/libssl/ApplicationProtocolContext.cs

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace System.Net
+{
+    internal enum ApplicationProtocolNegotiationStatus
+    {
+        None = 0,
+        Success,
+        SelectedClientOnly
+    }
+
+    internal enum ApplicationProtocolNegotiationExt
+    {
+        None = 0,
+        NPN,
+        ALPN
+    }
+
+    internal class ApplicationProtocolContext
+    {
+        public ApplicationProtocolNegotiationStatus NegotiationStatus = ApplicationProtocolNegotiationStatus.None;
+        public ApplicationProtocolNegotiationExt NegotiationExtension = ApplicationProtocolNegotiationExt.None;
+
+        public string GetProtocolId()
+        {
+            return null;
```

Comment:
I assume this is just a temporary stub until the functionality is actually implemented on Unix?  is there an issue tracking it?